### ISSUE TITLE
[VK Video] "Unknown error" when downloading playlist's JSON metadata

### ISSUE THREAD

**Issue by iG8R** [yt-dlp/yt-dlp#6219]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a broken site
- [X] I've verified that I'm running yt-dlp version **2023.01.06** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Ukraine, but not limited to.

### Provide a description that is worded well enough to be understood

While trying to download video's playlist, e.g. https://vk.com/video/playlist/-179915138_101 , an error appears.

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-vU', '--skip-download', 'https://vk.com/video/playlist/-179915138_101']
[debug] Encodings: locale cp65001, fs utf-8, pref cp65001, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2023.01.06 [6becd25] (win_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.19045-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: ffmpeg n5.1.2-9-g807afa59cc-20221210 (setts), ffprobe n5.1.2-9-g807afa59cc-20221210
[debug] Optional libraries: Cryptodome-3.16.0, brotli-1.0.9, certifi-2022.12.07, mutagen-1.46.0, sqlite3-2.6.0, websockets-10.4
[debug] Proxy map: {}
[debug] Loaded 1760 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: 2023.01.06, Current version: 2023.01.06
yt-dlp is up to date (2023.01.06)
[vk:uservideos] Extracting URL: https://vk.com/video/playlist/-179915138_101
[vk:uservideos] -179915138_101: Downloading webpage
[download] Downloading playlist: Становление богом
[vk:uservideos] -179915138: Downloading JSON metadata
ERROR: Unknown error
Traceback (most recent call last):
  File "yt_dlp\YoutubeDL.py", line 1502, in wrapper
  File "yt_dlp\utils.py", line 3053, in <lambda>
  File "yt_dlp\utils.py", line 2823, in __getitem__
  File "yt_dlp\extractor\vk.py", line 563, in _entries
  File "yt_dlp\extractor\vk.py", line 82, in _download_payload
yt_dlp.utils.ExtractorError: Unknown error

[vk:uservideos] Playlist Становление богом: Downloading 1 items of 0
[download] Finished downloading playlist: Становление богом
```

**Comment by dirkf** [comment]
`Unknown error` is what the site returns. Possibly the `load_videos_silent` payload API used in #2248 is no longer working.

**Comment by bashonly** [comment]
This looks like it will be fixed by #6122
